src/sentry/static/sentry/app/components/forms/selectControl.jsx

```diff
@@ -148,6 +149,19 @@ const defaultStyles = {
   clearIndicator: indicatorStyles,
   dropdownIndicator: indicatorStyles,
   loadingIndicator: indicatorStyles,
+  groupHeading: provided => ({
+    ...provided,
```

Comment:
Should you spread after the defaults so that the defaults could be customized if necessary?

Reply:
Are the defaults in this case the styles I added such as:
`lineHeight: 1.5`, `fontWeight: 600`, etc..? If so those are customized by passing in a custom style prop to the `SelectControl` component. Do correct me if I'm misunderstanding something though.